src/Invite/Invite.model.js

```diff
@@ -0,0 +1,25 @@
+import { Model } from 'decentraland-commons'
+import { SQL } from '../database'
+
+export class Invite extends Model {
+  static tableName = 'decentraland_invites'
+  static columnNames = ['address', 'invited']
+
+  static createOrUpdate(address, invited) {
+    return Invite.query(
+      SQL`INSERT INTO ${SQL.raw(this.tableName)} (address, invited) 
+          VALUES (${address}, ${invited})
+          ON CONFLICT (address) DO UPDATE
+          SET invited = EXCLUDED.invited`
+    )
+  }
+
+  static async hasInvite(address) {
+    const res = await this.findByAddressAndStatus(address, true)
+    return res !== undefined
+  }
+
+  static findByAddressAndStatus(address, invited) {
+    return this.findOne({ address, invited })
+  }
+}
```

Comment:
Do you think `findByAddressAndStatus ` will be used differently as `hasInvite` use it?

Maybe we can just have  ~`hasInvite(address)`~ `hasInvitation` or `isInvited` 

```javascript
static async isInvited(address) {
    const res = await this.findOne({ address, invited: true })
    return res !== undefined
  }
```

Reply:
ok